src/pages/Protocol.vue

```diff
@@ -1,233 +0,0 @@
-<template>
-	<www-page>
-		<div id="banner-container">
-			<div id="banner-content" class="tw-bg-primary">
-				<p id="banner-text" class="tw-text-h1">
-					Building the credit bureau of the future
-				</p>
-				<div id="banner-logo">
-					<img id="banner-logo-image" src="~@/assets/images/protocol/kiva-protocol.svg">
-				</div>
-			</div>
-		</div>
-		<div class="row" id="intro">
-			<div class="small-12 columns tw-prose">
-				<p id="intro-title" class="tw-text-subhead">
-					{{ introTitle }}
-				</p>
-				<p v-for="(paragraph, index) in introText" :key="index">
-					{{ paragraph }}
-				</p>
-				<p><a href="https://pages.kiva.org/kiva-protocol-faq">{{ introLinkTitle }}</a></p>
-			</div>
-		</div>
-		<div id="support-container">
-			<div class="row">
-				<div id="support-donate-container" class="small-12 large-6 columns">
-					<h2 id="support-title">
-						Support Kiva innovation
-					</h2>
-					<div id="support-text">
-						{{ supportText }}
-					</div>
-					<kv-custom-donation :default-value="100" />
-				</div>
-				<div class="show-for-large medium-8 medium-offset-2 large-5 large-offset-1 columns">
-					<kv-responsive-image
-						:images="billionImpactImages" alt="10 years and $1 billion in loans funded"
-					/>
-				</div>
-			</div>
-		</div>
-		<div id="about-container" class="tw-bg-secondary">
-			<div class="row" id="about">
-				<div class="small-12 large-7 columns">
-					<h2>
-						About Kiva
-					</h2>
-					<p id="about-text">
-						{{ aboutText }}
-					</p>
-				</div>
-				<div
-					id="charity-navigator-container"
-					class="small-8 small-offset-2 large-4 large-offset-1 columns"
-				>
-					<kv-charity-navigator />
-				</div>
-			</div>
-		</div>
-	</www-page>
-</template>
-
-<script>
-import WwwPage from '@/components/WwwFrame/WwwPage';
-import KvCharityNavigator from '@/components/Kv/KvCharityNavigator';
-import KvCustomDonation from '@/components/Kv/KvCustomDonation';
```

Comment:
Looks like KvCustomDonation isn't used anywhere except this page and can probably be removed too.